### PR TITLE
Misc. updates for new repositories and releasing stuff

### PR DESCRIPTION
**Pull request by aiyengar2**
Used for the RKE2 charts migration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,20 @@ charts-build-scripts
 
 ## Before running the scripts
 
-#### If you are creating a new charts repository with `charts-build-scripts init`
+#### If you are creating a new charts repository 
 
-Set the environment variable `GITHUB_AUTH_TOKEN` to your [personal Github Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
+Checkout into the Git branch that corresponds to your Source, Staging, or Live branch.
```
Comment (cbron) [inline README.md:10]: nit: "Checkout into the" -> "Checkout the"

File: templates/template/scripts/regenerate-packages [hunk 2]
```diff
@@ -3,6 +3,13 @@ set -e
 
 cd $(dirname $0)
 
+if [[ -z ${BRANCH} ]]; then
+    branch=$(git rev-parse --abbrev-ref HEAD)
+else
+    echo "Using branch ${BRANCH}"
```
Comment (cbron) [inline templates/template/scripts/regenerate-packages:9]: Would think this echo should be outside of conditional so it always prints ?

File: init.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+if [ "${BRANCH_ROLE}" != "source" ] && [ "${BRANCH_ROLE}" != "staging" ] && [ "${BRANCH_ROLE}" != "live" ] && [ "${BRANCH_ROLE}" != "custom" ]; then
+    echo "usage: BRANCH_ROLE=<source|staging|live|custom> ./init.sh"
+    exit 1
+fi
+
+# cd into the Git repository that you are converting into a charts repository
```
Comment (cbron) [inline init.sh:9]: this comment looks out of place

File: init.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    if [ "${BRANCH_ROLE}" = "source" ]; then
+        echo "2. Modify .github/workflows/pull-request.md and .github/workflows/push.md to set up automatic pushes to another branch."
+    fi
+else
+    echo "Creating an empty configuration.yaml file."
+    echo -n "" > configuration.yaml
+    echo "You will need to run make docs manually after filling in the configuration.yaml"
+    echo "To add a template for Github Workflow based pull-requests, run the following script and update .github/workflows/pull-request.yaml manually"
+    echo "curl ${CBS_RAW_LINK}/templates/.github/workflows/pull-request.yaml --output .github/workflows/pull-request.yaml"
+     echo "To add a template for Github Workflow based automatic pushes, run the following script and update .github/workflows/push.yaml manually"
```
Comment (cbron) [inline init.sh:39]: nit: extra space

**Review by cbron — COMMENTED** [review]
Don't fully comprehend the init.sh stuff, but I get what you are going for

File: README.md (hunk 1 above)
Comment (aiyengar2) [reply README.md:10]: will update, good catch.

File: templates/template/scripts/regenerate-packages (hunk 2 above)
Comment (aiyengar2) [reply templates/template/scripts/regenerate-packages:9]: Sure, will update.

File: init.sh (hunk 3 above)
Comment (aiyengar2) [reply init.sh:9]: ack, I left that behind on accident. Removed.

File: init.sh (hunk 4 above)
Comment (aiyengar2) [reply init.sh:39]: ack, updated

**Comment by aiyengar2** [comment]
Closing this in favor of https://github.com/rancher/charts-build-scripts/pull/24, which is based off of my fork.

@cbron, I've addressed all the comments here in the other PR.

**Pull request closed without merging**